src/status_im/chat/events/commands.cljs

```diff
@@ -68,3 +75,12 @@
      {:dispatch [:request-permissions {:permissions [:read-external-storage]
                                        :on-allowed  #(re-frame/dispatch [:initialize-geth])}]}
      (log/debug "ignoring command: " command-name))))
+
+;; NOTE(goranjovic) - continues execution of a command that was paused by a shortcut
+(handlers/register-handler-fx
+ :execute-stored-command
+ [re-frame/trim-v]
```

Comment:
Minor point, but as we don't use the event arguments at all, so we can omit interceptors (`re-frame/trim-v` in this case) a use function signature `(fn [{:keys [db]} _])`

Reply:
Right, this was a copy paste from above.